### ISSUE TITLE
请问fluid中怎么使用sgd+adam这种优化方式

### ISSUE THREAD

**Issue by Kayven** [PaddlePaddle/Paddle#14703]
在v2中这样的trainer.SGD设置update_equation，但fluid不支持

adam_optimizer = paddle.optimizer.Adam(                                                                                                       
    learning_rate=1e-4,                                                                                                                       
    regularization=paddle.optimizer.L2Regularization(rate=1e-3),                                                                              
    model_average=paddle.optimizer.ModelAverage(average_window=0.5))                                                                          
                                                                                                                                                                                                                                                                
trainer = paddle.trainer.SGD(                                                                                                                 
    cost=cost,                                                                                                                                
    extra_layers=paddle.evaluator.auc(input=prob, label=label),                                                                               
    parameters=parameters,                                                                                                                    
    update_equation=adam_optimizer, is_local=True)

**Comment by xuezhong** [comment]
在fluid中，sgd和Adam都是optimizer，V2版本里面这个地方最终调用的也是adam，所以等价地，你应该在fluid中直接调用adam这个optimizer就行
